docs/src/main/asciidoc/trace.adoc

```diff
@@ -97,7 +95,8 @@ spring.sleuth.web.skipPattern=(^cleanup.*|.+favicon.*)  # Ignore some URL paths.
 Spring Cloud GCP Trace does override some Sleuth configurations:
 
 - Always uses 128-bit Trace IDs. This is required by Stackdriver Trace.
-- Does not use Span joins. Span joins will share the span ID between the client and server Spans. Stackdriver requires that every Span ID within a Trace to be unique, so Span joins are not supported.
+- Does not use Span joins. Span joins will share the span ID between the client and server Spans.
```

Comment:
One more period in this line to break it into new line.

Reply:
Nice catch.